docs/organizing-and-exploring-results-in-the-ui/single-experiment-details.rst

```diff
@@ -0,0 +1,138 @@
+Exploring a single experiment
+=============================
+
+|Youtube Video|
+
+Click a line in the experiments table to see details of that experiment.
```

Comment:
[consider replacing]

> Click a line in the experiments table to see details of that experiment.

Click on the experiment ID (leftmost column) to enter single experiment view.